packages/core/src/editor/BlockNoteEditor.ts

```diff
@@ -113,7 +113,7 @@ export type BlockNoteEditorOptions<
    * @param file The file that should be uploaded.
    * @returns The URL of the uploaded file OR an object containing props that should be set on the file block (such as an id)
    */
-  uploadFile: (file: File) => Promise<string | Record<string, any>>;
+  uploadFile: (file: File) => Promise<string | Record<string, any> | void>;
```

Comment:
these changes are wrong I think? `uploadFile` shouldn't be able to be void

Reply:
Resolved